src/types/Item.ts

```diff
@@ -0,0 +1,22 @@
+import { Description } from './Description';
+
+export interface Item {
+  id: string,
+  namespaceId: string,
+  name: string,
+  capacityAvailable: string[],
+  capacity: string,
+  priceRegular: number,
+  priceDiscount: number,
+  colorsAvailable: string[],
+  color: string,
+  images: string[],
+  description: Description[],
+  screen: string,
+  resolution: string,
+  processor: string,
+  ram: string,
+  camera: string,
+  zoom: string,
```

Comment:
`camera` and `zoom` fields are absent in `accessories` section. You may want to make these optional